packages/runtime-common/realm.ts

```diff
@@ -165,6 +166,28 @@ const CACHE_HIT_VALUE = 'hit';
 const CACHE_MISS_VALUE = 'miss';
 const MODULE_ETAG_VARIANT = 'module';
 const SOURCE_ETAG_VARIANT = 'source';
+const DEFAULT_FILE_CONTENT_TYPE = 'application/octet-stream';
+const FILE_DEF_CODE_REF: ResolvedCodeRef = {
+  module: `${baseRealm.url}file-api`,
+  name: 'FileDef',
+};
+const CONTENT_TYPE_BY_EXTENSION: Record<string, string> = {
+  '.json': 'application/json',
+  '.md': 'text/markdown',
+  '.txt': 'text/plain',
+  '.csv': 'text/csv',
+  '.tsv': 'text/tab-separated-values',
+  '.ts': 'text/typescript',
+  '.js': 'text/javascript',
+  '.gts': 'text/x-gts',
+  '.png': 'image/png',
+  '.jpg': 'image/jpeg',
+  '.jpeg': 'image/jpeg',
+  '.gif': 'image/gif',
+  '.svg': 'image/svg+xml',
+  '.webp': 'image/webp',
+  '.pdf': 'application/pdf',
+};
```

Comment:
there's an npm package that can do this https://www.npmjs.com/package/mime-types which is actually already part of our project.